final_prep/annotations.rb

```diff
@@ -3,41 +3,41 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# defining method build_a_bear
 def build_a_bear(name, age, fur, clothes, special_power)
-  greeting = "Hey partner! My name is #{name} - will you be my friend?!"
-  demographics = [name, age]
-  power_saying = "Did you know that I can #{special_power}?"
-  built_bear = {
-    'basic_info' => demographics,
-    'clothes' => clothes,
-    'exterior' => fur,
-    'cost' => 49.99,
-    'sayings' => [greeting, power_saying, "Goodnight my friend!"],
-    'is_cuddly' => true,
+  greeting = "Hey partner! My name is #{name} - will you be my friend?!" # string filling in a greeting with name argument interpolated
+  demographics = [name, age] # giving name, age to demographics in an array
+  power_saying = "Did you know that I can #{special_power}?" #giving a saying that includes special_power argument
+  built_bear = { # the bear being described/ properties of the bear in a hash
+    'basic_info' => demographics, # providing name and age key
+    'clothes' => clothes, # what the bear is wearing key
+    'exterior' => fur, # describing that the exterior is fur key
+    'cost' => 49.99, # cost of bear key
```

Comment:
Good job here. I may additionally point out whether the value comes from a variable declaration above, or is being passed into the build_a_bear method.